### PR TITLE
Added style as a props

### PR DESCRIPTION
**Pull request by shruthidasarapu**
# Description

Added style as aprop so that everyone can change according to their requirement
Fixes # (issue)
image

# How to test?
npm run storybook

Please provide a short summary how your changes can be tested?
Everyone can change the styling
# Checklist

- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/Button/Button.js [hunk 1]
```diff
@@ -2,12 +2,12 @@ import React from "react";
...
-  return <button className= "button" type="submit" style= {{backgroundColor}}> {buttonName} </button>;
+const Button = ({buttonName,style}) => {
+  return <button className= "button" type="button" style= {style}> {buttonName} </button>;
 };
 export default Button;
  
 Button.propTypes = {
   buttonName: PropTypes.string.isRequired,
-  backgroundColor:PropTypes.string.isRequired
+  style: PropTypes.string.isRequired
```
Comment (pdsorensen) [inline src/client/components/Button/Button.js:12]: I think it should be optional. It's an edge case if you want to style the button. The primary styles should reside in the button component itself as i see it. :)

File: src/client/components/Button/Button.stories.js [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+    backgroundColor: 'orange',
+    border: '1px solid black',
+    borderRadius: 4,
+    fontSize: 15,
+    }
+  return <Button buttonName ={text("Label", "Primary")} style={object('Style', style)}  />;
 };
 export const Secondary = () => {
-  return <Button buttonName = {text("Label", "Secondary")} backgroundColor = {color("backgroundColor", "blue")}  />;
+  const style = {
```
Comment (pdsorensen) [inline src/client/components/Button/Button.stories.js:20]: i would move these styles into the css.

File: src/client/components/Button/Button.js (hunk 1 above)
Comment (shruthidasarapu) [reply src/client/components/Button/Button.js:12]: ok, should i remove style from here

Comment (pdsorensen) [reply src/client/components/Button/Button.js:12]: No, you should make it optional. Right now it's required. :)

Comment (pdsorensen) [reply src/client/components/Button/Button.js]: @shruthidasarapu should'nt it also be a object? And i think the default value should just be an empty object.

Comment (shruthidasarapu) [reply src/client/components/Button/Button.js]: i tried with object but it shows error 
image

Comment (pdsorensen) [reply src/client/components/Button/Button.js]: Allright, hm we still have a failing check. Could you see if you can fix that? :)

File: src/client/components/Button/Button.styles.css [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-    padding: 15px 32px;
-    text-align: center;
-    text-decoration: none;
-    font-size: 16px;
-    border-radius: 4px;
-    cursor: pointer;
-    display: inline-block;
-}
\ No newline at end of file
+  color: white;
```
Comment (pdsorensen) [inline src/client/components/Button/Button.styles.css:2]: Hm, is there any reason why we need to change the original styles?

Comment (shruthidasarapu) [reply src/client/components/Button/Button.styles.css:2]: Nothing.  I just choose  the default

**Review by pdsorensen — APPROVED** [review]

**Pull request merged**